chore: update segment source doc page (#471)
knocklabs/docs: content/integrations/sources/segment.mdx

  }
/>

<Steps>
  <Step title="Add webhook destination in Segment"> 
    In your Segment workspace, navigate to the **Destinations** tab and click "Add Destination." From here, search for **Webhooks** or navigate to **Raw Data** in the sidebar and click the "Webhooks (Actions)" option. This is the destination type we'll use for Knock.
    ![Configure webhooks](/images/configure-webhooks.png)
  </Step>
  <Step title="Configure webhook">
    Click the "Configure Webhooks (Actions)" button in the top right to enter the setup flow to configure your webhook:
    <AccordionGroup>
      <Accordion title="Select data source">
        Check the data source you'd like to receive events from and click "Next."
      </Accordion>
      <Accordion title="Setup">
        Give a name to the destination, like "Knock", and keep the "Fill in settings manually" option selected, then click "Create destination."
      </Accordion>

>     </AccordionGroup>

Author: @cellomatt I've also been on the fence about the best practice for accordions. They are really nice at organizing the page, but I do agree that sometimes it doesn't make sense.

I just pushed up an update without _without_ the accordions, which you can view here: https://docs-ip9rkfnua-knocklabs.vercel.app/integrations/sources/segment
And then this is the old version _with_ the accordions: https://docs-djfqltido-knocklabs.vercel.app/integrations/sources/segment

I lean towards removing them, but let me know your thoughts!

>     </AccordionGroup>

Reviewer: Oh thanks, that's really helpful to visualize!

I think in this case I also like it better without. I'll keep this bullet-point alternative in mind, it could be the better approach in some situations

>     </AccordionGroup>

Author: Cool, I'll go ahead and get this merged in then 🎉 

Yeah I think with time we will have a better feel for the best practices for the components and can feel confident about ironing out the style guide 💪

  </Step>
  <Step title="Save changes">
    Now you'll see a **Basic Settings** page. Check the "Enable Destination" button and click "Save changes."
  </Step>
  <Step title="Add mapping">
    At the top of the page, navigate to the **Mappings** tab. Click "New Mapping", and under Actions, click "Send".

>     You will then be directed to a form to configuring the webhook:

Reviewer: ```suggestion
    You will then be directed to a form to configure the webhook:
```

## Disabling an action

> Actions are automatically enabled when you create them. To stop an event from triggering an action, go to the action page and toggle its status to **Inactive.** Keep in mind that this will disable that action for the current environment only. When you're ready to trigger the action again, you can set it back to **Active.**

Reviewer: Personal preference nit:
```suggestion
Actions are automatically enabled when you create them. To stop an event from triggering an action, go to the action page and toggle its status to **Inactive.** Keep in mind that this will only disable that action for the current environment. When you're ready to trigger the action again, you can set it back to **Active.**
```

    }}
  ></iframe>
</div>

## Frequently asked questions

<AccordionGroup>

>   <Accordion title="Can I have one event trigger multiple workflows?">

Reviewer: I feel like this could be worded slightly differently but I'm not sure exactly what about it bugs me; feel free to take it or leave it :) 
```suggestion
  <Accordion title="Can I use a single Segment event to trigger multiple workflows?">
```

    Yes. Under the **Developers** > **Sources** section in your Knock dashboard,
    select an event from Segment to view a list of actions configured for that
    event. Then, in the upper right-hand corner, click the "Create action"
    button to select the additional workflow you want this event to trigger,
    then click "Create." You can then make any changes to the schema mapping

>     before saving and committing this new action to your development

Reviewer: ```suggestion
    before saving and committing this new action to your Development
```

    environment.
  </Accordion>

>   <Accordion title="Can I have multiple events trigger one workflow?">

Reviewer: Similar to above, take it or leave it:
```suggestion
  <Accordion title="Can I use multiple different Segment events to trigger the same workflow?">
```

>     Yes. To do so, you will follow the steps above to create an action for the

Reviewer: Just calling out that if the above accordion is closed, "the steps above" might not be fully obvious
